src/content/en/2020/seo.md

```diff
@@ -83,183 +84,227 @@ Response Code | All Devices | Mobile | Desktop
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=1777344456&format=interactive",
   sheets_gid="337739550",
   sql_file="pages_wpt_bodies_by_device.sql"
- )
-}}
+) }}
 
-When analyzing this year’s mobile home pages canonical tag configuration, it was detected that 45.31% of them were self-referential and 8.45% were pointing to different URLs as the canonical ones.
+When analyzing this year's mobile home pages canonical tag configuration, it was detected that 45.31% of them were self-referential and 8.45% were pointing to different URLs as the canonical ones.
 
+{# TODO(authors): Consider omitting desktop-specific interpretations unless they're particularly interesting to note beyond the mobile stats. #}
 On the other hand, 51.85% of the desktop home pages were found to be featuring a canonical tag this year, with 47.88% being self-referential and 4.10% pointing to a different URL.
 
-Not only mobile home pages are featuring more canonical tags than desktop ones (53.61% vs. 51.85%) but there are more mobile homes pages canonicalizing to other URLs than their desktop counterparts (8.45% vs. 4.10%). This could be explained by the usage of an independent (or separate) mobile web version by some sites -that need to canonicalize to their desktop URLs alternates.
+Not only do mobile home pages include more canonical tags than desktop ones (53.61% vs. 51.85%), there are relatively more mobile homes pages canonicalizing to other URLs than their desktop counterparts (8.45% vs. 4.10%). This could be explained by the usage of an independent (or separate) mobile web version by some sites that need to canonicalize to their desktop URLs alternates.
 
-Canonical URLs can be specified through different methods, by using the canonical link via the HTTP headers or the HTML head of a page, or by submitting them in XML Sitemaps. When analyzing which is the most popular canonical link implementation method, it was found that only 1.03% of desktop homepages and 0.88% of mobile ones are relying on the HTTP headers for their implementation, meaning that canonical tags are prominently implemented via the HTML head of a page.
+Canonical URLs can be specified through different methods: by using the canonical link via the HTTP headers or the HTML `head` of a page, or by submitting them in XML sitemaps. When analyzing which is the most popular canonical link implementation method, it was found that only 1.03% of desktop home pages and 0.88% of mobile ones are relying on the HTTP headers for their implementation, meaning that canonical tags are prominently implemented via the HTML `head` of a page.
 
+{# TODO(analysts): Should "HTTP Head" be "HTTP Header" to distinguish it from HTML head? #}
```

Comment:
@Tiggerito

Reply:
@antoineeripret another chart edit. HTTP Head to HTTP Header. 

https://docs.google.com/spreadsheets/d/1ram47FshAjzvbQVJbAQPgxZN7PPOPCKIK67VJZCo92c/edit#gid=337739550

Last chart.

Reply:
Updated and new PNG file to be added in the commit. 

@Tiggerito @rviscomi